sdk/keyvault/keyvault-certificates/src/lro/recover/operation.ts

```diff
@@ -65,7 +67,9 @@ async function update(
     try {
       state.result = await client.getCertificate(certificateName, { requestOptions });
       state.isCompleted = true;
-    } catch (_) {}
+    } catch (e) {
```

Comment:
Does this `e` not lint as unused? I would have expected it to, but maybe `catch` is handled a little differently.

Reply:
yep, catch is handled differently. Though I wonder why underscore isn't valid, when TypeScript says it's valid.

Reply:
TS has a rule that will ignore _-named parameters. Presumably eslint doesn't have a similar one.

FWIW, if ESLint has support, you could just eliminate the `(e)` part entirely. I believe this was added in TS 3.6.

Reply:
Ohh! Good to know! Thank you!!!!

Reply:
:( eslint doesn't like the empty `()` on that catch part. It says:

```
  70:13  error  Parsing error: Identifier expected
```

Reply:
I'll merge this for now, but I'll catch up if I'm missing anything with another PR.